solutions/swb-reference/integration-tests/tests/isolated/sshKeys/get.test.ts

```diff
@@ -71,7 +54,7 @@ describe('Get Key Pair negative tests', () => {
     });
     test('it throws 404 error', async () => {
       try {
-        await adminSession.resources.projects.project(invalidProjectId).sshKeys().get();
+        await paabResources.adminSession.resources.projects.project(invalidProjectId).sshKeys().get();
```

Comment:
This test will still need more rewriting after middleware comes in, but this refactor is good in that it adjusts to the current change without finishing the test.

After middleware comes in, I am pretty sure an invalid project id will be a 403 since dynamic authz will say "I don't see permissions for that project for this user!" because the project doesn't have _any_ permissions registered since it does not exist. Let's keep as is for now though.